src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -554,8 +555,9 @@ void TwoPointFluxApproximation::initNewFractureFieldsDFM( MeshLevel & mesh,
     }
     fluidPressure_n[newElemIndex] = fluidPressure[newElemIndex];
 #if ALLOW_CREATION_MASS==0
-    // set the initial density of the face element to 0 to enforce mass conservation ( i.e. no creation of mass)
-    dens[newElemIndex] = 0.0;
+    // set the initial mass/density of the face element to 0 to enforce mass conservation ( i.e. no creation of mass)
```

Comment:
@CusiniM not sure about this piece, it does not seem even compile when ALLOW_CREATION_MASS is 0

Reply:
i think in the fracture propagation, it provides different options to initialize the new element. At least in FP, new cells are always filled with fluid once generated.

Reply:
yes, it does offer different options but it was done through macros. This option was basically never used. This should be fixed in https://github.com/GEOS-DEV/GEOS/pull/2960 which we should review and merge.

Reply:
Is it ok if merge it like that for now and refactor after https://github.com/GEOS-DEV/GEOS/pull/2960 is merged?